core/src/main/java/org/opentosca/toscana/core/transformation/TransformationDao.java

```diff
@@ -19,18 +20,19 @@
      * @param csar     the csar on which the new transformation will be based on
      * @param platform the target platform of the new transformation
      * @return the new transformation
+     * @throws UnsupportedPlatformException if given platform is not supported by any known plugin
      */
-    Transformation create(Csar csar, Platform platform);
+    Transformation create(Csar csar, Platform platform) throws UnsupportedPlatformException;
```

Comment:
Does the frontend (rest api) need to handle this exception?

Reply:
yes - look at the patch in TransformationController